### DISCUSSION TITLE
clangd cannot resolve libassert's <assert.hpp> in header file

### DISCUSSION THREAD

**Discussion by smac89** [clangd/clangd#2017] [Q&A]
As the header says, when I include <assert.hpp> in a header file, clangd cannot seem to find the header. However, when I do the same in a `.cpp` file, it is resolved without an issue.
The program compiles fine even with the error clangd shows.

How do I configure clangd to find the right header?

**Comment by HighCommander4** [comment]
Have you followed the steps at https://clangd.llvm.org/installation#project-setup ?
